support/certs/tls.go

```diff
@@ -356,11 +369,20 @@ func ReconcileSignedCert(
 		secret.Data[caKey] = append([]byte(nil), ca.Data[opts.CASignerCertMapKey]...)
 	}
 
+	var certValidity = ValidityOneYear
+
+	if os.Getenv(CertificateValidityEnvVar) != "" {
+		customCertValidityEnvVar, err := time.ParseDuration(os.Getenv(CertificateValidityEnvVar))
+		if err != nil {
+			certValidity = customCertValidityEnvVar
+		}
+	}
```

Comment:
```suggestion
	if value := os.Getenv(CertificateValidityEnvVar); value != "" {
		customCertValidityEnvVar, err := time.ParseDuration(value)
		if err != nil {
			return fmt.Errorf("invalid validity duration: %v", err)
		}
		certValidity = customCertValidityEnvVar
	}
```

Reply:
Talking with @csrwng we agree to avoid fail when the duration is not well set to avoid interfere with the normal workflow.

Reply:
Please fail if we cannot parse the duration.